docs/snippets/install-module.ps1

```diff
@@ -1,6 +1,5 @@
 Set-PSRepository -Name PSGallery -InstallationPolicy Trusted
 Install-Module -Name VMware.PowerCLI -MinimumVersion 13.0.0
 Install-Module -Name PowerVCF -MinimumVersion 2.3.0
-Install-Module -Name PowerValidatedSolutions -MinimumVersion 2.4.0
```

Comment:
Does this module no longer required PowerValidatedSolutions ?

Reply:
I've confirmed with @joisika and @garlicNova that there is no dependency on `PowerValidatedSolutions`. There may be future code efficiencies we could pull into the module if we want to add the depedency.